tutorials/continuous_aggs_tutorial.md

```diff
@@ -0,0 +1,173 @@
+
+# Tutorial: Continuous Aggregates
+
+
+A continuous aggregate query computes the query results automatically at a user specified time interval without the user having to repeatedly rerun the query.  This is particularly useful when recomputing aggregates frequently for large data sets.
+
+We will explore this feature using the data set from the [Hello NYC Tutorial][hello_nyc]
+
+Prerequisites:
+TimescaleDB installed (version 1.3 or greater)
+
+### 1. Download and Load Data
+
+Let's start by downloading the dataset.
+
+This dataset contains two files:
+1. `contagg_nyc_data.sql` - A SQL file that will set up the necessary tables for the continuous aggregates tutorial.
+1. `nyc_data_rides.csv` - A CSV file with the ride data
+
+First, create a database, e.g., `nyc_data` with the extension:
+
+```sql
+CREATE DATABASE nyc_data;
+\c nyc_data
+CREATE EXTENSION IF NOT EXISTS TimescaleDB CASCADE;
+```
+
+Next, download the file from the below link:
+
+[:DOWNLOAD_LINK: `nyc_data.tar.gz`][nyc_data]
```

Comment:
There is no corresponding link ref at bottom

Reply:
fixed